### PR TITLE
Completed String manipulation 2 - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-04-03 at 6 18 47 PM

### CODE REVIEW

File: string-manipulation-2/src/capitalize-word.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+function capitalizeWord(string) {
+  var divide = string.split(' ');
+  var empty = '';
+  for (var i = 0; i < divide.length; i++) {
+    if (divide[i].toLowerCase() === 'javascript') {
+      return 'JavaScript';
+    } else {
+      empty += divide[i][0].toUpperCase() + divide[i].slice(1).toLowerCase();
+    }
+  }
```
Comment (codingcodymiller) [inline string-manipulation-2/src/capitalize-word.js:11]: This is for a single word, there should be no need for a loop

File: string-manipulation-2/src/capitalize-words.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  var k = [];
```
Comment (codingcodymiller) [inline string-manipulation-2/src/capitalize-words.js:3]: Please stop using single letters for your variable names, with the exception of `i`. Your variables should be named to describe their purpose, _always_

File: string-manipulation-2/src/truncate.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/* exported truncate */
+function truncate(length, string) {
+  var nstring = '';
```
Comment (codingcodymiller) [inline string-manipulation-2/src/truncate.js:3]: `nstring`? What does this mean?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here, please see my notes

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**